mysystehis/main.tex

```diff
@@ -0,0 +1,1155 @@
+ \begin{figure}[h]
+ 	\begin{minipage}{1\textwidth}
+ 		\centering
+ 		\includegraphics[width=0.8\linewidth]{intructions/light_categlories.png}
+ 		\centering
+ 		\end{minipage}
+ 	\end{figure}
+ \\[0.05cm]
+ In computer graphic, this act of light bouncing around into two categories: direct illumination and indirect illumination. Direct lighting is when light gets send out from a light source, bounces and then reaches the eye (or the camera). Indirect lightning is when light bounces off multiple surfaces before hitting the eye, light will keep bouncing until it get fully absorbed. From the figure 26, all the type of light from Light section, like directional light, point light, etc.. they are all direct lights because they're all light source. When defining about indirect lighting, which mean every kind of light that's not directly point anywhere from the scenes, but by going to Window -> Rendering -> Lighting and look at environment lighting section, that's a great example of indirect lighting and so GI. And this's actually when GI basically comes to play, and GI will take care of direct lighting and indirect lighting that reflect from surfaces meaning it helps illuminating the scene more realistically. 
+ \begin{figure}[h]
+ 	
+ \end{figure}
+ Without GI enabled, i cannot see any light bounce at all, even the light source is directly pointing at the sphere. With GI enabled, the room is now become a bit brighter , with a bit of light bouncing to the ceiling. The way the GI works, is that when the light travel through the gap that seen from the image above, and then the photon that travel from the light source will carry the values of surface that hit to the next surface. And with that way, that's can explain the reason why i can see the green light on the ceiling, that's the light that inherit from the green cube and blue sphere. 
+  \begin{figure}[h]
+ 	\begin{minipage}{1\textwidth}
+ 		\centering
+ 		\includegraphics[width=0.8\linewidth]{intructions/How_light_works.png}
+ 		\centering
+ 	\end{minipage}
+ \end{figure}
+\\[0.05cm]
+ In unity, it provides two techniques for pre computing GI: Baked GI and pre-computed real-time GI. The Baked one is pre-computed and will save a ton of processing power, and this only work if all the objects, light-source in the scene are static, and real-time is extractly lighting calculated in real-time, per frame, means that the light, shadow position can be changed overtime when an object or the light source is moving. whenever to change the mode to baked or real-time, this depend on each object, light source. For the sunlight (the light that shines the whole scene), i set the mode to real-time because there're still some moving objects like the player, the NPC, etc... and for the light like the room light, i set it to baked mode. A combination of baked and realtime lights can help maintain performance
```

Comment:
....light-sources in the scene are static. Whereas real-time is exactly lighting calculated in real-time, per frame, which means light and shadow positions can change over time when an object or light source is moving. Depending on each object and light source, we select the appropriate GI method.